marconi-chain-index/changelog.d/20230224_061323_konstantinos.lambrou_PLT_776_fix_test_running_time_of_tx_address_to_utxo_address_test.md

```diff
@@ -0,0 +1,11 @@
+### Changed
```

Comment:
I think we should decide on what goes into the changelog and what not, currently it looks more like things that usually go into a commit message now go into changelog instead, and if someone wants to correlate commits and changes one would now have to do something like `git log --patch -- marconi-chain-index/changelog.d` to get a human language description of what changes there were.

Also I don't know if the users of marconi also be interested in if linter and compiler warnings were fixed or if the test now runs faster.

Are writing it for ourselves or for somebody else?

Reply:
Totally agree. I **do** use as a commit message :D. I would even argue we don't write changelogs until a version is officially released.